daemon/daemon_unix.go

```diff
@@ -407,6 +422,12 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 			return warnings, fmt.Errorf("Cannot use the --read-only option when user namespaces are enabled.")
 		}
 	}
+	if hostConfig.CgroupParent != "" && daemon.usingSystemd() {
```

Comment:
This change LGTM.

While you are here, can you also update container_operations_unix.go to perform the same validation, as well as use the new usingSystemd() function?